script/tool/lib/src/analyze_command.dart

```diff
@@ -64,6 +65,14 @@ class AnalyzeCommand extends PluginCommand {
     }
 
     final List<Directory> packageDirectories = await getPackages().toList();
+    final List<String> packagePaths =
+        packageDirectories.map((Directory dir) => dir.path).toList();
```

Comment:
`toSet()`? You're only using this for inclusion testing.

Reply:
will change